lib/sidekiq/instrument/middleware/server.rb

```diff
@@ -17,15 +17,21 @@ def call(worker, job, _queue, &block)
       execution_time_ms = (Time.now - start_time) * 1000
       Statter.statsd.measure(metric_name(worker, 'runtime'), execution_time_ms)
       Statter.dogstatsd&.timing('sidekiq.runtime', execution_time_ms, worker_dog_options(worker))
-    rescue StandardError => e
+    rescue Exception => e
```

Comment:
Rescuing `Exception` is not recommended ([ref](https://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby)), so what's the reason for this change?

Reply:
Per the description:

> Changes rescue StandardError => e to now be rescue Exception => e to properly handle gathering stats for all potential error types. The error is re-raised at the end of the rescue so there is no need for concern in widening the umbrella of what we're catching.

We're missing error types without capturing `Exception` and not just `StandardError`. I'll send you the investigation ticket for context.